benchmarks/ZAP/how_to_run_zap.md

```
@@ -0,0 +1,141 @@
+## Prerequisites
+
+| Tool | Version | Notes |
+|---|---|---|
+| Docker | 20+ |
+| Python | 3.9+ |
+
+---
+
+## Overview
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)